### PR TITLE
Community Article - Troubleshooting MySQL with MySQLTuner

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a56c5a555f/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-97f21d7411/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-7f7dcd7145/how-to/). :bow:

**Comment by cyrichardson** [comment]
@erikmwilson Suggestions here: 
[troubleshooting-mysql-configuration-with-mysqltuner-script_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2877581/troubleshooting-mysql-configuration-with-mysqltuner-script_cyrEdits.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5f435fd77f/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b189dcfcab/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ad820098f4/how-to/). :bow:

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+permalink: troubleshoot-mysql-configuration-with-mysqltuner-script
```
Comment (RackerKate) [inline content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: 1. Add a forward slash (/) at the end of the permalink.
2. Change the file name to match the permalink.

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+audit_date: '2019-01-18'
+title: Troubleshoot MySQL configuration with MySQLTuner script
```
Comment (RackerKate) [inline content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Suggest "...with a MySQLTuner script." If you update this, suggest updating the file name to match.

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+created_date: '2019-01-18'
+created_by: Rackspace Community
+last_modified_date: '2019-01-19'
```
Comment (RackerKate) [inline content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Change to 2019-2-20.

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+last_modified_by: '2019-01-18'
```
Comment (RackerKate) [inline content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Change to your name.

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md (hunk 1 above)
Comment (erikmwilson) [reply content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Done

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md (hunk 2 above)
Comment (erikmwilson) [reply content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Done

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md (hunk 3 above)
Comment (erikmwilson) [reply content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Done

File: content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md (hunk 4 above)
Comment (erikmwilson) [reply content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Done

Comment (erikmwilson) [reply content/cloud-servers/troubleshooting-mysql-with-mysqltuner.md]: Done

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e50cddd7d7/how-to/)

**Pull request merged**